Add include field to package-references (#4644)
ballerina-platform/ballerina-dev-website: learn/guides/organize-ballerina-code/package-references.md

├── modules/
│     ├── model/
│     └── module1.test/

> ├── documents/

Reviewer: Shall we change this to have one directory and one file in the package root

Author: Updated to have,
├── documents/
├── sample.png

name = "winery"
version = "0.1.0"
export = ["winery", "winery.model"]
include = ["documents", "sample.png"]

> icon = “icon.png”

Reviewer: The symbol for `"` here seems to be wrong

*   Once the package is production-ready, you can use a stable version (E.g. 1.0.0). Any subsequent minor or patch releases of the same major version should be backward compatible and, should not break existing builds.

### The `export` field

You can specify the modules which should be visible to the outside using the `export` field. The export field accepts a string array, which contains the names of the modules which
need to be publicly visible.

> The modules should be referred in the format <pacakge_name>.<module_name>. The default module should be referenced by the package name.

Reviewer 2: ```suggestion
The modules should be referred in the `<pacakge_name>.<module_name>` format. The default module should be referenced by the package name.
```

### The `icon` field

The `icon` field accepts a path to an icon. The specified icon will be packaged into the `docs/` directory of the BALA.

> Only the .png format is supported for the icon.

Reviewer 2: ```suggestion
Only the `.png` format is supported for the icon.
```

*   Once the package is production-ready, you can use a stable version (E.g. 1.0.0). Any subsequent minor or patch releases of the same major version should be backward compatible and, should not break existing builds.

### The `include` field

> You can provide paths to any additional resources that needs to be packed in the BALA file during the use of `bal pack` command.

Reviewer 2: ```suggestion
You can provide paths to any additional resources, which need to be packed in the BALA file during the use of the `bal pack` command.
```

> The include field is an array that accepts relative paths from the root of the package to the files or directories. 

Reviewer 2: ```suggestion
The `include` field is an array, which accepts relative paths from the root of the package to the files or directories. 
```

> The include field is an array that accepts relative paths from the root of the package to the files or directories. 

Reviewer: The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The paths should be relative to the package root directory.

> The included files and directories will have a file structure similar to the orginal package from the root of the BALA file.

Reviewer: The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The paths should be relative to the package root directory. The included file paths will be packaged into the root of the BALA preserving its original structure.
